akka-persistence/src/main/scala/akka/persistence/AtLeastOnceDelivery.scala

```diff
@@ -128,6 +128,32 @@ trait AtLeastOnceDeliveryLike extends Eventsourced {
     Persistence(context.system).settings.atLeastOnceDelivery.redeliverInterval
 
   /**
+   * Constance used for multiplication interval after delivery fail
+   *
+   * The default value can be configure with the
+   * `akka.persistence.at-least-once-delivery.redeliver-factor`
+   * configuration key. This method can be overridden by implementation classes to return
+   * non-default values
+   */
+  def redeliverFactor: Int = defaultRedeliverFactor
+
+  private val defaultRedeliverFactor: Int =
+    Persistence(context.system).settings.atLeastOnceDelivery.redeliverFactor
+
+  /**
+   * Constance used to define random part od redelivery delay.
+   *
+   * The default value can be configure with the
+   * `akka.persistence.at-least-once-delivery.redeliver-random-part`
+   * configuration key. This method can be overridden by implementation classes to return
+   * non-default values.
+   */
+  def redeliverRandomPart: Double = defaultRedeliverRandomPart
+
+  private val defaultRedeliverRandomPart: Double =
+    Persistence(context.system).settings.atLeastOnceDelivery.redeliverRandomPart
```

Comment:
These things will unfortunately not be binary backwards compatible, since it's a trait. Didn't we discuss that in the ticket?